onnx/defs/operator_sets.h

```diff
@@ -636,6 +636,7 @@ class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 11, Pad);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 11, Gemm);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 11, If);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 11, NonMaxSuppression);
+class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 11, Adam);
```

Comment:
Maybe I misunderstood, but I thought opset 11 is already part of release 1.6. Should the new operator be in opset 12 for release 1.7?

Reply:
Sorry.. This commit is for merging with master branch. Please only review the last commit 2d28250.